crt_portal/cts_forms/templates/partials/attachment-uploading-modal.html

```diff
@@ -0,0 +1,19 @@
+{% load i18n %}
+
+<div id="attachment-uploading--modal" hidden>
+  <div class="modal-wrapper" role="dialog" aria-modal="true" aria-describedby="modal_dialog_header">
+    <div class="modal-content modal-content--small">
+      <div class="modal-header">
+        <h1 id="modal_dialog_header" class="h3__display">
+          {% trans "Uploading..." %}
+        </h1>
+      </div>
+      <div class="modal-form">
+        <p>
+          {% trans "Your attachment is uploading..." %}
+        </p>
+        <input type="hidden" />
```

Comment:
Ahh, what if we used `tabindex` to make the `h1` focusable here instead of adding an input? 

Since openModal will focus on the `h1` when called, a screenreader will reader the Uploading content and give folks non-visual feedback as well as the modal.

Reply:
Yes- wasn't thinking about accessibility! Thanks - making this change w/ Imran now